docs/Dependencies.md

```diff
@@ -162,9 +162,17 @@
 
    1. `CybSDK_app`: you can download the application from [this link](https://developer.cyberith.com/downloads). Go to `Tools` tab, download the `Virtualizer Control Panel`. Copy the download application to the workspace. Everytime start using the virtualizer, please calibrate it using this application.
      
-   2. `CybSDK`: you can download the application from [this link](https://developer.cyberith.com/downloads). Go to `SDK` tab, download the `C++ SDK for Windows (Developed for: Visual Studio 2015 Community)`. Copy the download application to the workspace. in theory, we should build it in our Windows machine, but time being it is already built (there exists the .dll file) and is working, so you do not need to build it.
+   2. `CybSDK`: you can download the application from [this link](https://developer.cyberith.com/downloads). Go to `SDK` tab, download the `C++ SDK for Windows (Developed for: Visual Studio 2015 Community)`. Copy the download application to the workspace.
    
-      - Append the following variables to the User environmental variable `Path`:
+     - **If you are using version later than [v0.2.0](https://github.com/robotology/walking-teleoperation/tree/v0.2.0)**: After Downloading the SDK, append the following variables to the user environmental variable:
```

Comment:
In principle, you can avoid this kind of comments if you are changing directly the README of the 2.0 version. I guess it is ok also like this at the moment. We just have to remember to remove this comment once ``devel`` gets merged in ``master``.

Reply:
Actually, you are right. Also, I should have opened the PR on top of `devel`, not the `master`. So, I will update the PR.

Reply:
@S-Dafarra  updated, please have a look.